sw/host/rom_ext_image_tools/src/main.rs

```diff
@@ -2,6 +2,22 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
```

Comment:
I don't know if we're compiling this with `-D warnings`, but just in case, you should include these pragmas in this file:
```
#![deny(warnings)]
#![deny(missing_docs)]
#![deny(unused)]
#![deny(unsafe_code)]
```